app/controllers/CapiProxyController.scala

```diff
@@ -2,32 +2,20 @@ package controllers
 
 import com.gu.contentapi.json.CirceEncoders._
 import io.circe.syntax._
-import play.api.libs.json.JsValue
 import play.api.mvc._
-import play.api.libs.json.{Json, Reads, Writes}
 import services.{ContentClient}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-object CapiSearchParams {
-  implicit val writes: Writes[CapiSearchParams] = Json.writes[CapiSearchParams]
-  implicit val reads: Reads[CapiSearchParams] = Json.reads[CapiSearchParams]
-}
-
-case class CapiSearchParams(query: String, tags: Option[List[String]] = None, sections: Option[List[String]] = None)
-
 class CapiProxyController(cc: ControllerComponents, contentClient: ContentClient)(implicit ec: ExecutionContext)
   extends AbstractController(cc) {
 
-  def searchContent: Action[JsValue] = Action.async(parse.json) { request =>
-    request.body.validate[CapiSearchParams].asEither match {
-      case Right(query) => contentClient.searchContent(
-          query.query,
-          query.tags.getOrElse(Nil),
-          query.sections.getOrElse(Nil))
-        .map { result => Ok(result.asJson.toString).as(JSON) }
-      case Left(error) => Future.successful(BadRequest(s"Invalid request: $error"))
-    }
+  def searchContent(query: String, tags: Option[List[String]], sections: Option[List[String]]) = Action.async {
```

Comment:
@aug24 This change moves the search parameters for the CAPI client from the request body to url params; Play will do the parsing and typechecking for us, and it's a better fit for what should be a GET request.

This will be useful in upcoming work, but there's no particular reason to include the change in this PR, so if we'd like it moved out then please shout 👍